archive/brave-web-servlet-filter/src/main/java/com/github/kristofa/brave/util/UriEscaperUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.github.kristofa.brave.util;
```

Comment:
move to internal package and fine by me src//main/java/com/github/kristofa/brave/servlet/internal

Reply:
Done